applications/CoSimulationApplication/python_scripts/coupling_interface_data.py

```diff
@@ -113,6 +118,12 @@ def GetBufferSize(self):
     def InplaceMultiply(self, factor):
         self.SetData(factor*self.GetData())
 
+    def AddToData(self, data_to_add):
```

Comment:
Shouldn't such operations be as a DataTransferOperations ? That makes the Data class simpler !

Reply:
Or overload __add__ function so we can directly call data3 = data1 + data2

Reply:
https://www.geeksforgeeks.org/operator-overloading-in-python/

Reply:
I will remove it
Unfortunately implementing the `__add__` operator does not work bcs this would mean creating Modelparts on the fly, which is not possible